samples/bot-initiate-thread-in-channel/csharp/TeamsApp/teamsapp.local.yml

```diff
@@ -0,0 +1,85 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:bot-initiate-thread-in-channel-csharp
+
+provision:
+  - uses: aadApp/create # Creates a new Azure Active Directory (AAD) app to authenticate users if the environment variable that stores clientId is empty
```

Comment:
It seems `botAadApp/create` is more popular for bot aad registration, but if it can work fine, I am also good to it.
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/15262146/9b7e645c-dfda-4095-92de-043b279ceb32)